frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.scss

```diff
@@ -2,6 +2,10 @@
   margin-bottom: var(--pf-global--spacer--md);
 }
 
+.ocs-storageClass__form {
```

Comment:
Let's use this commons CSS class for all OCS storage class extension to maintain a  uniform bottom space, Updating thick provision PR also with the same CSS class